blogs.json

```diff
@@ -5323,6 +5323,14 @@
             "author": "Zachary Shakked",
             "site_url": "https://blog.zach.so",
             "feed_url": "https://blog.zach.so/feed"
+          },
+          {
+            "title": "Swifty Ship",
+            "author": "Valentin Mille",
+            "site_url": "https://valentinmille.com/newsletter/",
```

Comment:
Thanks for this @valentin-mille! Is it intentional that the links for the site and RSS feed are the same for both entries? If you had two independent publications then it'd be fine to add them both but with the same content in each I'd ask you to pick one section you feel they fit best in.

Thanks!

Reply:
Thanks for your prompt feedback. It was intentional, I send the newsletter each week and it appears as a blog post after that. I just removed the one in blogs to keep the newsletter one :)